src/map/MapVEuMap.tsx

```diff
@@ -156,6 +156,10 @@ export interface MapVEuMapProps {
   showNoDataOverlay?: boolean;
   /** Whether to show the Scale in the map */
   showScale?: boolean;
+  /** mouseMode control */
+  mouseMode: 'default' | 'magnification';
```

Comment:
We could save some typing/maintenance with
```suggestion
  mouseMode: MouseMode;
```